### PR TITLE
Rahti CI CD introduction

### PR DESCRIPTION
**Pull request by tdeneke**

### CODE REVIEW

File: docs/cloud/rahti/tutorials/ci_cd_introduction.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+Make sure you are in the right project with `oc project` and if not you can create one 
+with `oc new-project <your-new-project-name>`. 
+
+We have all the necessary object definitions in the `rahti-bc-example` project under the `k8s-api-objs.yaml`. 
+Please refer to [Kubernetes and OpenShift concepts](/cloud/rahti/concepts/) to understand the objects 
+defined under `k8s-api-objs.yaml`. You can easily create these objects necessary for our CI/CD pipeline using 
+the `oc create` command as follows:
+
+```
+oc create -f k8s-api-objs.yaml
```
Comment (lvarin) [inline docs/cloud/rahti/tutorials/ci_cd_introduction.md:53]: nitpick add a `$` for the commands, and leave the output like it is.

File: docs/cloud/rahti/tutorials/ci_cd_introduction.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+will be to kick off the build and let the pipeline handle the build and 
+deployment. You can trigger such an action using the `oc build` as follows:
+
+```
+oc start-build dockerfile-example --from-dir=./ -F
+```
+
+Once the pipeline finishes running you can visit your application deployment 
+by following the route which should look like `http://dockerfile-example-<your_project_name>.rahtiapp.fi/`.
+This pipeline can be triggered again after any update to your application (e.g. update on `index.html`) and
```
Comment (lvarin) [inline docs/cloud/rahti/tutorials/ci_cd_introduction.md:71]: You should add here a link to the web-hooks tutorial page. There is a link at the end of this document, but I do not think the intended audience will know.

**Review by lvarin — COMMENTED** [review]
Looks good, the only two comments are details

File: docs/cloud/rahti/tutorials/ci_cd_introduction.md (hunk 2 above)
Comment (tdeneke) [reply docs/cloud/rahti/tutorials/ci_cd_introduction.md:71]: Done.

File: docs/cloud/rahti/tutorials/ci_cd_introduction.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+Make sure you are in the right project with `oc project` and if not you can create one 
+with `oc new-project <your-new-project-name>`. 
+
+We have all the necessary object definitions in the `rahti-bc-example` project under the `k8s-api-objs.yaml`. 
+Please refer to [Kubernetes and OpenShift concepts](/cloud/rahti/concepts/) to understand the objects 
+defined under `k8s-api-objs.yaml`. You can easily create these objects necessary for our CI/CD pipeline using 
+the `oc create` command as follows:
+
+```
+$oc create -f k8s-api-objs.yaml
```
Comment (lvarin) [inline docs/cloud/rahti/tutorials/ci_cd_introduction.md:53]:
```suggestion
$ oc create -f k8s-api-objs.yaml 
```

**Review by lvarin — APPROVED** [review]
LGTM

**Pull request merged**